codebase/superdarn/src.bin/tk/tool/make_raw.1.8/make_raw.c

```diff
@@ -214,11 +219,27 @@ int main (int argc,char *argv[]) {
     exit(-1);
   }
 
+  command[0]=0;
```

Comment:
Can you explain in the comments what is going on here?

Reply:
```suggestion
 // Transfer command line arguments into "command" array (max 128 characters)
  command[0]=0;
```
@egthomas Is the `command[0]=0;` necessary? The code seems to work without it but I may have missed something here.

Reply:
In this pull request I've simply added the same code to handle the origin fields as is currently used in all of the other binaries (eg, [dattorawacf](https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.bin/tk/reformat/dattorawacf.1.8/dattorawacf.c#L172), [fittofitacf](https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.bin/tk/reformat/fittofitacf.1.8/fittofitacf.c#L156), [make_fit](https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.bin/tk/tool/make_fit.1.19/make_fit.c#L314), [make_fitex](https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.bin/tk/tool/make_fitex.1.4/make_fitex.c#L158), etc).  If someone wants to go back and update all of them to be consistent as far as comments or removal of certain lines then I don't see any problem.

Reply:
@egthomas if that is how you feel about commenting that is fine I did it for you. I figured this PR was about consistency to being well documented on that consistency would be good.